docs/source/movielens.rst

```diff
@@ -25,6 +25,10 @@ Format MovieLens 100K Item Metadata Data
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 .. autofunction:: collie.movielens.get_movielens_metadata
```

Comment:
This doesn't have to be in this PR, but I wonder if we should deprecate this function to be more consistent with the user version below, something like ``collie.movielens.get_item_metadata``. What do you think?